developer/src/common/web/test-helpers/verifyCompilerMessagesObject.ts

```diff
@@ -0,0 +1,66 @@
+import { CompilerErrorSeverity, compilerErrorSeverityName } from '@keymanapp/common-types';
+import {assert, expect} from 'chai';
+
+//
+// The purpose of these tests is to ensure that our slightly WET constant
+// definitions for compiler errors are consistent in their usage:
+//
+//   * that names are consistent between the message functions and constants
+//   * that constants have unique codes
+//   * that constants have the right error mask for their type
+//
+// This means we don't have to worry about DRYing out the constants, which would
+// probably require us to have a preprocessing step, which has its own
+// negatives.
+//
+// These are used in each of the test-messages.ts units to verify the
+// CompilerMessages classes
+
+const toTitleCase = (s: string) => s.substring(0, 1).toUpperCase() + s.substring(1).toLowerCase();
```

Comment:
Doesn't handle Greek titlecase, of course https://unicode-org.github.io/icu/userguide/transforms/general/rules.html#combinations

Reply:
Good news then that I don't plan to use Greek (or German) characters in my compiler message identifiers!